src/action-creators/cursorDown.js

```diff
@@ -8,18 +8,17 @@ import {
   isDivider,
   restoreSelection,
   selectNextEditable,
+  getThoughts
 } from '../util.js'
 
 export const cursorDown = ({ target }) => dispatch => {
-
   const { cursor } = store.getState()
-
   if (cursor) {
     // select next editable
     const nextThought = getThoughtAfter(cursor)
-
+    const children = getThoughts(cursor)
     if (nextThought) {
-      const nextThoughtsRanked = contextOf(cursor).concat(nextThought)
+      const nextThoughtsRanked = children && children.length > 0 ? cursor.concat(children[0]) : contextOf(cursor).concat(nextThought)
```

Comment:
You can just check `children.length > 0 ?` since we know that `children` is an array.